### PR TITLE
Add notification sound setting

### PR DESCRIPTION
**Pull request by sienna-kopf**
# SOM-Timer BE Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
- add sound column to timers table as a new setting. 

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Has This Been Tested?
- [ ] No
- [x] Yes

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
- updated test for getting one timer and updating a timer to include sound field 
- all tests passing
- checked in Postman as well.

# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs/pry sessions
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by Abdeboskey — APPROVED** [review]
Migrated, seeded, and tested just like that?!?!?! I am amazed.

**Review by rwilliams659 — APPROVED** [review]
Sienna, thanks so much for the quick updates!!! Super appreciated. It would be awesome if you could also update the documentation to reflect this change when you have a minute.

File: application/controllers/timers.py [hunk 1]
```diff
@@ -30,5 +31,8 @@ def put(self, timer_id=None):
         if 'rest_interval' in request.json:
             timer.rest_interval = request.json['rest_interval']
 
+        if 'sound' in request.json:
+            timer.sound = request.json['sound']
+
         db.session.commit()
```
Comment (Chulstro) [inline application/controllers/timers.py:37]: Well and thorough thoughts on what all needs to change the timer table

**Review by Chulstro — APPROVED** [review]
Looks good!

**Pull request merged**